opensearch-operator/controllers/cluster_test.go

```diff
@@ -66,6 +67,14 @@ var _ = Describe("Cluster Reconciler", func() {
 				return true
 			}, timeout, interval).Should(BeTrue())
 		})
+
```

Comment:
didnt understand that test

Reply:
Hey @idanl21 this will actually match the resources declared in [suite_test_helpers.go](https://github.com/Opster/opensearch-k8s-operator/pull/69/files/9eaf0382da39c1f602345174b33c16abdcaafa47#diff-49f5e716774e22cfecc7184a15d7a3407c6afcee1e6e5e5898471d850b5efe27)